server/athenian/api/internal/account.py

```diff
@@ -347,12 +349,20 @@ async def copy_teams_as_needed(account: int,
     created_teams = []
     for node_id in reversed(list(nx.topological_sort(dig))):
         team = teams[node_id]
-        if (parent := teams.get(team[MetadataTeam.parent_team_id.name])) is not None:
-            parent = db_ids[parent[MetadataTeam.id.name]]
+
+        parent_id = None
+        if (meta_parent_id := team[MetadataTeam.parent_team_id.name]) is not None:
+            if (parent := teams.get(meta_parent_id)) is not None:
+                parent_id = db_ids[parent[MetadataTeam.id.name]]
+
+        if parent_id is None:
```

Comment:
```suggestion
        github_parent_id = team[MetadataTeam.parent_team_id.name]
        try:    
            parent_id = db_ids[teams[github_parent_id][MetadataTeam.id.name]]
        except KeyError:
            parent_id = root_team_id
```

Reply:
for me here it is less readable to try `teams[None]` when github team has no parent

Reply:
I changed to this version in https://github.com/athenianco/athenian-api/pull/2392
```python
parent_id = root_team_id
if (github_parent_id := team[MetadataTeam.parent_team_id.name]) is not None:
    if (parent := teams.get(github_parent_id)) is not None:
        parent_id = db_ids[parent[MetadataTeam.id.name]]
```